doc/dev/Mgmt-Polishing-Configurations.md

```diff
@@ -0,0 +1,355 @@
+
+# Guidelines for Data Type Changes
+
+In the API specifications, many properties and parameters are defined as string types but have special formats in .NET. We need to correct these types in the Azure .NET SDK.
+
+- The data type of a property should be `uuid` if its name is end of `Id` / `Guid` and value is really a uuid.
+
+- The data type of a property should be `ResourceIdentifier` if its name is end of `Id` and value is really a ResourceIdentifier.
+
+- The data type of a property should be `ResourceType` if its name is `ResourceType` or end with `Type` and the value is really a ResourceType('Microsoft.xxxxx/xxxx').
+- The data type of a property should be `ETag` if its name is `etag` and the value is really a ETag. The property name  should be `ETag`(Optional).
+
+- The data type of a property might be `AzureLocation` if its name is end with `location` / `locations` .
+
+- `Binary` / `String` property might change to `BinaryData` type.
+
+- Pay attention to models similar to common type models and confirm whether they really can't be replaced by the common types from ResourceManager(Optional). 
+
+# Guidelines for Operation Changes
+
+- Put / patch operation, optional body parameter should be changed to required.
+
+- All `ListOperations` method should be removed because it's not useful in SDK and the SDK itself can show all the available operations in it through public APIs. 
+
+# Configuration for Generating SDK from TypeSpec Specifications
+For SDKs generated from TypeSpec specifications, we can update the client.tsp file located in the same folder as the main.tsp to address APIView review comments in the .NET SDK.
+
+## Using the correct namespace
+
+Before starting, please ensure that the correct namespace is used at the beginning of the client.tsp file, as shown below:
+
+```js
+using Microsoft.DeviceRegistry;
+```
+
+You can typically find the namespace in the main.tsp file.
+
+## Rename models
+
+To change the name of a model, use the [@clientname](https://azure.github.io/typespec-azure/docs/libraries/typespec-client-generator-core/reference/decorators/#@Azure.ClientGenerator.Core.clientName) decorator as shown below:
+
+```js
+@@clientName(Asset, "DeviceRegistryAsset", "csharp");
+```
+
+This renames the name of model `Asset` to `DeviceRegistryAsset`.
+
+## Rename properties
+
+To change the name of a properties, use the [@clientname](https://azure.github.io/typespec-azure/docs/libraries/typespec-client-generator-core/reference/decorators/#@Azure.ClientGenerator.Core.clientName) decorator as shown below:
+
+```js
+@@clientName(ExtendedLocation.type, "ExtendedLocationType", "csharp");
+```
```

Comment:
I think these are quire fundamental knowledge for typespec, do we really need to cover all of them here (including the above using namespace statement).
I think maybe we could just reference the doc for `@clientName` decorator - or other useful decorators here

Reply:
We should not expect all people know typespec well.